datasets/doc/source/conf.py

```diff
@@ -47,6 +49,8 @@
 extensions = [
     "sphinx.ext.napoleon",
     "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx_autodoc_typehints",
```

Comment:
Should we add these to other docs projects as well?

Reply:
I use the auto-summary in the project to generate links to classes and methods like here <img width="812" alt="image" src="https://github.com/adap/flower/assets/51029327/852ce9c2-e4a3-45d6-9994-53bd76339163">

Reply:
I will delete the `sphinx_autodoc_typehints`. I didn't finally use it.